### ISSUE TITLE
Line ending error occurs when starting up nvim

### ISSUE THREAD

**Issue by yang-ling** [wsdjeg/SpaceVim#3995]
## Expected behavior

SpaceVim should startup without error.

## Actual behavior

SpaceVim shows below error when starting up

```
Error detected while processing /home/yangling/.SpaceVim/bundle/vim-jsx-typescript/ftdetect/typescript.vim:
line    5:
E492: Not an editor command: ^M
```

## Environment Information

- OS: linux
- vim version: -
- neovim version: 0.4.4
- SpaceVim version: 1.6.0-dev
- SpaceVim status: 6e25213d

```
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

## The reproduce ways from Vim starting (Required!)

1. Start `nvim`

## Screenshot

[![asciicast](https://asciinema.org/a/zfCZtgh6muLJeqxJ8469V8z93.svg)](https://asciinema.org/a/zfCZtgh6muLJeqxJ8469V8z93)

## Cause and possible fix

- It is `bundle/vim-jsx-typescript/ftdetect/typescript.vim` in commit 6e25213d3ef55043de5b0379e09e3d1515ed1480 which causes this bug.
- Changing its line ending to unix style fixes this bug.

**Comment by hakamadare** [comment]
fixed by #3998

**Comment by wsdjeg** [comment]
Ok, closed.
